src/ontology/vbo-odk.yaml

```diff
@@ -21,6 +21,8 @@ import_group:
       module_type: custom
     - id: omo
       module_type: mirror
+    - id: cob
+      module_type: mirror
```

Comment:
not sure if mirror is necessary.. Did Chris tell you this? I would have though default (SLME BOT) is fine?

Reply:
Chris didn't say anything, I did what we did for omo. So there is no good reason to have it.